administrator/language/en-GB/com_config.ini

```
@@ -95,6 +95,8 @@ COM_CONFIG_FIELD_FORCE_SSL_DESC="HTTPS must be enabled on your server or load ba
 COM_CONFIG_FIELD_FORCE_SSL_LABEL="Force HTTPS"
 COM_CONFIG_FIELD_GZIP_COMPRESSION_DESC="Joomla can automatically compress the generated HTML pages with GZip, making them smaller and increasing your site's speed score. Disable it if your server is already doing that for you or if it conflicts with third party extensions."
 COM_CONFIG_FIELD_GZIP_COMPRESSION_LABEL="Gzip Page Compression"
+COM_CONFIG_FIELD_HELPURL_DESC="The default is https://help.joomla.org/proxy?keyref=Help50:{keyref}&lang={langcode}"
+COM_CONFIG_FIELD_HELPURL_LABEL="URL of Help server"
```

```suggestion
COM_CONFIG_FIELD_HELPURL_LABEL="URL of Help Server"
```

please follow the style guide